shelley/chain-and-ledger/executable-spec/src/STS/Utxo.hs

```diff
@@ -95,7 +95,7 @@ utxoInductive = do
   ups' <- trans @(UP hashAlgo dsignAlgo) $ TRC ((slot_, pp, dms_), u ^. ups, txup tx)
 
   let outputCoins = [c | (TxOut _ c) <- Set.toList (range (txouts txBody))]
-  all (0<) outputCoins ?! NonPositiveOutputsUTxO
+  all (0 <=) outputCoins ?! NonPositiveOutputsUTxO
```

Comment:
nitpick: should we then also rename the error slightly, e.g., `NegativeOutputsUTxO` ?

Reply:
yes!